docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -580,6 +580,7 @@ The Nessie catalog does not support [view management](sql-view-management) or
 
 ### Snowflake catalog
 
+Snowflake catalog is a read-only catalog. Additionally, Snowflake doesn't support partitioned Iceberg tables.
```

Comment:
We should move the limitation down and explain more .. similar to how the MV and view support limitation is documented in the others.

Reply:
sure, updated. Please review.